README.md

```diff
@@ -56,8 +76,92 @@ make lint
 make gen-docs
 ```
 
+### Debugging
+
+To debug the plugin in Intellij so that code can be stepped through with breakpoints, you will need:
+1. [Delve](https://github.com/go-delve/delve) debugger installed.
+2. A run/debug configuration set up in Intellij.
+To set this up, add an Intellij run/debug configuration:
+    1. `Run` > `Edit Configurations...` > `+` > `Go Remote`
+    2. Fill in `Host: localhost` and `Port: 7777`
+3. A local spec file that specifies a local gRPC process as the plugin:
+```yaml
```

Comment:
Would it be possible/useful to check these spec files in, say in a 'run locally' directory or something, rather than the user having to create them every time?

Reply:
The danger is I suppose someone filling it in and then accidentally committing it - so may be better just to have something here in the readme that someone could copy and paste.